modules/serverless-sinkbinding-odc.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-odc_{context}"]
+= Create a SinkBinding using the Developer perspective
+
+This guide describes the steps required to create, manage, and delete a sample SinkBinding instance by using **Developer** perspective in the {product-title} web console.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have switched to the *Developer* perspective in the web console.
+
+.Procedure
+
+. Create the `event-display` sample Service to use as an event sink.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Container Image**.
++
+image::serverless-odc-sinkbinding-1.png[Add a sample Service event sink as a Container Image]
++
+.. For the **Image** configuration, select **Image name from external registry**, then enter the sample URL:
++
+----
+quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+// TODO: Needs an update to latest sample image for 1.8.0?
++
+The **General** fields, **Application** and **Name**, will be filled automatically once the sample image is provided.
+.. For **Resources**, select **Knative Service**.
+.. Click **Create**.
+. Create a SinkBinding.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Event Source**.
++
+image::serverless-sinkbinding-odc-2.png[Configuring SinkBinding]
++
+.. Select the **SinkBinding** icon.
+.. In the form, enter the following configuration:
+... For **apiVersion** enter `v1`.
+... For **Kind**, enter `SinkBinding`.
++
+image::serverless-sinkbinding-odc-3.png[Cofiguring a sink]
++
+... For **Sink** -> **Knative Service**, select your Service from the dropdown menu.
++
+The **General** fields, **Application** and **Name**, will be filled automatically once the sample image is provided.
++
+.. Click **Create**.
+
+.Verification steps
+
+. Navigate to **Topology** for the `knative-serving` **Project**.
+. You will see the SinkBinding connected to your Service:
```

Comment:
For other methods of creating _SinkBinding_ - [using kn CLI](https://docs.openshift.com/container-platform/4.5/serverless/knative_eventing/serverless-sinkbinding.html#serverless-sinkbinding-kn_serverless-sinkbinding), and [YAML methods](https://docs.openshift.com/container-platform/4.5/serverless/knative_eventing/serverless-sinkbinding.html#serverless-sinkbinding-yaml_serverless-sinkbinding), we also have verified that _SinkBinding_ is working well by creating some event producer and looking at container logs to confirm that.

Here we only look that ODC is displaying it right. It isn't the same confidence level verification method. Maybe here we also should confirm that it actually works?

Reply:
Yeah I can add that `oc` step.